octree/include/pcl/octree/octree_iterator.h

```diff
@@ -529,69 +530,68 @@ namespace pcl
      * \ingroup octree
      * \author Fabien Rozar (fabien.rozar@gmail.com)
      */
-    template<typename OctreeT>
-      class OctreeBreadthFirstAtDepthIterator : public OctreeBreadthFirstIterator<OctreeT>
-      {
-      public:
+      template<typename OctreeT>
```

Comment:
I probably was not clear enough. The `template` line was properly indented. But the `class` line and all following lines are shifted two spaces right with respect to it. So I asked to "unshift" them, i.e. remove two extra spaces.

Reply:
I hope I get it right now.